app/routers/users.py

```
@@ -2,23 +2,25 @@
 
 from __future__ import annotations
 
+import logging
 from typing import List
 
 from fastapi import APIRouter, Depends, HTTPException, Query, Response, status
 from sqlalchemy import select
+from sqlalchemy.exc import OperationalError
```

Flake8 (7.3.0)

[error] 'sqlalchemy.exc.OperationalError' imported but unused

(F401)